src/components/footer/index.js

```diff
@@ -55,6 +56,10 @@ function Footer() {
                 <p>
                     <a href="https://tarkovtracker.io/">TarkovTracker</a>
                 </p>
+                <h3>{t('Contributors')}</h3>
+                <p>Massive thanks to all the people who help build and maintain this project!</p>
+                <p>Made with ❤️ by:</p>
+                <Contributors quality={20} />
```

Comment:
I think it needs to be stacked by default, or show like 7-8 avatars (single line on mobile) because we have already too much stuff in the footer

Reply:
I actually think it looks heckin good on mobile

![image](https://user-images.githubusercontent.com/23362539/189266508-e163ae39-3674-42d1-a001-ebbe0e04275f.png)

Reply:
Let me know what you think but overall I think it looks good. The only issue with stacking is it can cause width overflows since we have a lot of contributors

Reply:
It looks good (but for my taste we still have too much stuff in the footer 🤣).

How does stacked work? Do you like click on it and it expands to not stacked?

Reply:
This is the Avatar Stack component, you can read about it here -> https://primer.style/react/AvatarStack

It has live examples as well. It works on a hover. However, when checking it out on our site, it seems to have weird borders around all the Avatars and I'm not sure why 🤔